docs/resources/private_endpoint_regional_mode.md

```diff
@@ -111,8 +137,8 @@ You can create only sharded clusters when you enable the regionalized private en
 ## Additional Reference
 
 In addition to the example shown above, keep in mind:
-* `mongodbatlas_cluster.cluster-atlas.depends_on` - Make your cluster dependent on the project's `mongodbatlas_private_endpoint_regional_mode` as well as any relevant `mongodbatlas_privatelink_endpoint_service` resources.  See an [example](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples/aws-privatelink-endpoint/cluster-geosharded). 
-* `mongodbatlas_cluster.cluster-atlas.connection_strings` will differ based on the value of `mongodbatlas_private_endpoint_regional_mode.test.enabled`.
+* `mongodbatlas_advanced_cluster.cluster-atlas.depends_on` - Make your cluster dependent on the project's `mongodbatlas_private_endpoint_regional_mode` as well as any relevant `mongodbatlas_privatelink_endpoint_service` resources.  See an [example](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples/aws-privatelink-endpoint/cluster-geosharded). 
```

Comment:
underscores are no valid in cluster names, see: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2445

Reply:
cc @maastha @EspenAlbert

Reply:
I'm not referring to the Atlas cluster name, it's the name of the Terraform `mongodbatlas_advanced_cluster` resource, so
`mongodbatlas_advanced_cluster.cluster_atlas.depends_on`
instead of `mongodbatlas_advanced_cluster.cluster-atlas.depends_on`

Reply:
Thank you!